joomla/joomla-cms
# [4.0] Adding accessibility settings to admin profile view

Pull Request for Issue #30217.

### Summary of Changes
The accessibility settings are missing from the profile view of com_admin. This copies those over from the com_users XML.


### Testing Instructions
In the backend in the upper right corner click on the user menu and select accessibility settings. Notice that the new page does not contain accessibility settings. Apply the patch, enjoy the new settings.
